### PR TITLE
remove padding from top issue#414

### LINKED ISSUE #414: [Bug]: Unnecessary top padding in button
### Describe the bug

Unnecessary top padding in button  , so text not able to align perfectly

### Steps To Reproduce

1.  Go to [home page](https://json-schema.org/)
2.  click to [Community page](https://json-schema.org/#community)
3.you can see [Read more](https://json-schema.org/blog/posts/understanding-lexical-dynamic-scopes) or "[View Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_8r4g9r3etmrmt83fm2gljbatos@group.calendar.google.com)"

### Expected Behavior

remove top margin

### Screenshots

Screenshot from 2024-03-02 20-57-58

Screenshot from 2024-03-02 20-57-43

### Device Information [optional]

```markdown
- OS:Linux Ubuntu 22.04.4 LTS
- Browser: Google Chrome
```

### Are you working on this issue?

Yes

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No

### PR DESCRIPTION
**Pull request by Ali7040**
**What kind of change does this PR introduce?**
Unnecessary top padding and block property.

**Issue Number:**
-  Closes #414 
-  Related to #___ 
-  Others? 

**Screenshots/videos:**

Screenshot 2024-03-03 at 7 53 06 PM

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

### CODE REVIEW

File: pages/index.page.tsx [hunk 1]
```diff
@@ -472,7 +472,7 @@ const Home = (props: any) => {
                 </div>
                 <a
                   href='https://calendar.google.com/calendar/u/0/embed?src=c_8r4g9r3etmrmt83fm2gljbatos@group.calendar.google.com'
-                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center pt-1 flex items-center justify-center'
+                  className='block w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center flex items-center justify-center'
```
Comment (DhairyaMajmudar) [inline pages/index.page.tsx:475]:
```suggestion
                  className='w-full lg:w-1/2 rounded border-2 bg-primary text-white  h-[40px] text-center flex items-center justify-center'
```

**Review by DhairyaMajmudar — APPROVED** [review]
LGTM : D 
Suggested some changes to resolve vs code warnings.

File: pages/index.page.tsx (hunk 1 above)
Comment (DhairyaMajmudar) [reply pages/index.page.tsx:475]: pls. also remove `leading-5` from line 242 reason same resolving vscode warnings.

**Review by benjagm — CHANGES_REQUESTED** [review]
Great job @Ali7040 . Can you please implement or resolve the suggestions made by @DhairyaMajmudar ?

**Comment by Ali7040** [comment]
> Great job @Ali7040 . Can you please implement or resolve the suggestions made by @DhairyaMajmudar ?

sure!

File: pages/index.page.tsx (hunk 1 above)
Comment (Ali7040) [reply pages/index.page.tsx]: Done.

**Comment by benjagm** [comment]
Thanks everyone! Is this ready to review again?

**Comment by Ali7040** [comment]
> Thanks everyone! Is this ready to review again?

yes!

**Pull request merged**